code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -270,7 +270,7 @@
 	id = "gygax_armor"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/part/gygax_armor
-	materials = list(MAT_METAL=25000,MAT_DIAMOND=10000)
+	materials = list(MAT_METAL=10000,MAT_DIAMOND=10000,MAT_TITANIUM=15000)
```

Comment:
I am seriously confused by your last message still
I didn't raise costs I replaced some metal and some of the other materials
Also I am going to have uses I'm just waiting for this to get merged before I can start those things cause git fucks you in the ass if you don't